### PR TITLE
Improve DOS plotting default axes ranges

### PR DESCRIPTION
**Pull request by mfherbst**

### CODE REVIEW

File: src/plotting.jl [hunk 1]
```diff
@@ -21,9 +21,21 @@ function ScfPlotTrace(plt=Plots.plot(yaxis=:log); kwargs...)
...
 
 
+function default_band_εrange(eigenvalues; εF=nothing)
+    if isnothing(εF)
+        # Can't decide where the interesting region is. Just plot everything
+        (minimum(minimum, eigenvalues), maximum(maximum, eigenvalues))
+    elseif is_metal(eigenvalues, εF)
+        εF .+ (-0.367, 0.367)
+    else
+        εF .+ (-0.147, 0.147)
```
Comment (antoine-levitt) [inline src/plotting.jl:31]: Where do these numbers come from? Are these ev? If so can you say explicitly? Also not 100% sure about the is metal stuff, how about just tracking whether there's a temperature or not?

Comment (mfherbst) [reply src/plotting.jl:31]: This is just what pymatgen does and this is just defaults for plotting. You can always change it later. I have no clue why they are doing it like that. Perhaps for metals you have more going on at the Fermi level so you care about a larger range. They are Hartree (in eV its 4 and 10)

Comment (antoine-levitt) [reply src/plotting.jl:31]: Just add a comment saying it's 4ev and it's stolen from pymatgen

Comment (mfherbst) [reply src/plotting.jl:31]: ok

**Pull request merged**